openmc/filter.py

```diff
@@ -719,21 +719,20 @@ def get_pandas_dataframe(self, data_size, summary=None):
 
         # energy, energyout filters
         elif 'energy' in self.type:
-            bins = self.bins
-            num_bins = self.num_bins
-
-            # Create strings for
-            template = '({0:.1e} - {1:.1e})'
-            filter_bins = []
-            for i in range(num_bins):
-                filter_bins.append(template.format(bins[i], bins[i+1]))
-
-            # Tile the energy bins into a DataFrame column
-            filter_bins = np.repeat(filter_bins, self.stride)
-            tile_factor = data_size / len(filter_bins)
-            filter_bins = np.tile(filter_bins, tile_factor)
-            filter_bins = filter_bins
-            df = pd.concat([df, pd.DataFrame({self.type + ' [MeV]' : filter_bins})])
+            # Extract the lower and upper energy bounds for each result.
+            lo_bins = self.bins[:-1]
+            hi_bins = self.bins[1:]
+
+            # Repeat and tile them as necessary to account for other filters.
+            lo_bins = np.repeat(lo_bins, self.stride)
```

Comment:
Can we compress these two lines to `repeat` with the two lines above? Something like:

```python
lo-bins = np.repeat(self.bins[:-1], self.stride)
...
```